app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.java

```diff
@@ -480,6 +482,17 @@ public void onComplete() {
                         });
 
                 break;
+            case 96:
+                // TODO: why does it break with v4?
```

Comment:
@nickvergessen same here, same API, so I is then just a marker for me :)

Reply:
Same above

Reply:
Fixed via https://github.com/nextcloud/talk-android/pull/1752/commits/0ba210a0cae90883395eb8aeec190a063831a567